contracts/Lockbox.cdc

```diff
@@ -0,0 +1,468 @@
+/*
+
+    Lockbox implements the functionality required to manage FLOW
+    buyers locked tokens from the token sale.
+
+    Each token holer gets two accounts. One account is their locked token
+    account. It will be jointly controlled by the user and Dapper Labs.
```

Comment:
Thinking about this again, it might be better to say "token administrator" rather than "Dapper Labs" -- eventually this will be controlled via governance so probably better to use a generic identifier.

Reply:
Good suggestion. I changed it to token administrator